pkg/collector/corechecks/oracle-dbm/statements.go

```diff
@@ -597,99 +562,32 @@ func (c *Check) StatementMetrics() (int, error) {
 				continue
 			}
 
-			startSQLText := time.Now()
 			queryRow := QueryRow{}
-			var queryHashCol string
 			var SQLStatement string
 
-			found := false
-			if statementMetricRow.StatementMetricsKeyDB.ForceMatchingSignature != "" {
-				obfuscatedStatement, ok := c.statementsCache.forceMatchingSignatures[statementMetricRow.StatementMetricsKeyDB.ForceMatchingSignature]
-				if ok {
-					queryRow.Commands = obfuscatedStatement.commands
-					queryRow.QuerySignature = obfuscatedStatement.querySignature
-					queryRow.Tables = obfuscatedStatement.tables
-					SQLStatement = obfuscatedStatement.statement
-					found = true
-				} else if c.config.InstanceConfig.QueryMetrics.IncludeDatadogQueries {
-					obfuscatedStatement, ok := c.DDPrevStatementsCache.forceMatchingSignatures[statementMetricRow.StatementMetricsKeyDB.ForceMatchingSignature]
-					if ok {
-						queryRow.Commands = obfuscatedStatement.commands
-						queryRow.QuerySignature = obfuscatedStatement.querySignature
-						queryRow.Tables = obfuscatedStatement.tables
-						SQLStatement = obfuscatedStatement.statement
-
-						c.DDstatementsCache.forceMatchingSignatures[statementMetricRow.StatementMetricsKeyDB.ForceMatchingSignature] = c.DDPrevStatementsCache.forceMatchingSignatures[statementMetricRow.StatementMetricsKeyDB.ForceMatchingSignature]
-						found = true
-					}
+			if statementMetricRow.SQLTextLength == 1000 {
+				getFullSQLText(c, &SQLStatement, "sql_id", statementMetricRow.RandomSQLID)
```

Comment:
The linter won't let you skip checking the error returned here. But I think it's also a good idea to handle the error.

Reply:
fixed now, thx/